README.md

```diff
@@ -1,7 +1,30 @@
+# Figma Plugin API typings
+[![npm](https://img.shields.io/npm/v/@figma/plugin-typings?logo=npm&cacheSeconds=1800)](https://www.npmjs.com/package/@figma/plugin-typings)
+
 This repository contains the typings for the Figma Plugin API.
 
-It is intended to be installed as "@figma/plugin-typings" and the types should become globally available without needing to use import statements. We do it this way because the plugin API is part of the host environment, as opposed to being a package that a plugin includes.
+## Usage
+
+1. Installation
+    ```sh
+    npm i --save-dev @figma/plugin-typings
+    # or
+    yarn add -D @figma/plugin-typings
+    ```
+
+2. Configure _tsconfig.json_
+
+    ```js
+    {
+        "compilerOptions": {
+            "typeRoots": [
+                "./node_modules/@types",
+                "./node_modules/@figma"
+            ]
+        }
+    }
+    ```
 
-The plugin will need to have a `tsconfig.json` which contains `"typeRoots": ["./node_modules/@types", "./node_modules/@figma"]` otherwise TypeScript won't look in the `node_modules/@figma/plugin-typings` folder by default.
```

Comment:
Adding an example is nice, but non-standard uses of tools (such as is the case here, where the typings don't come from DefinitelyTyped) warrant an explanation. It helps people debug if they messed up something, and the reader might leave having learned something new!